cdap-security/src/main/java/co/cask/cdap/security/server/LDAPAuthenticationHandler.java

```diff
@@ -35,19 +33,14 @@
   private static final List<String> mandatoryConfigurables = ImmutableList.of("debug", "hostname", "port", "userBaseDn",
                                                                               "userRdnAttribute", "userObjectClass");
 
-  private static final List<String> optionalConfigurables = ImmutableList.of("bindDn", "bindPassword", "useLdaps",
+  private static final List<String> optionalConfigurables = ImmutableList.of("bindDn", "useLdaps",
                                                                              "userIdAttribute", "userPasswordAttribute",
                                                                              "roleBaseDn", "roleNameAttribute",
                                                                              "roleMemberAttribute", "roleObjectClass");
-  private static boolean ldapSSLVerifyCertificate = true;
 
-  /**
-   * Create a new Authentication handler to use LDAP for external authentication.
-   */
-  @Inject
-  public LDAPAuthenticationHandler(CConfiguration configuration) throws Exception {
-    super(configuration);
-  }
+  private static final List<String> optionalSecureConfigurables = ImmutableList.of("bindPassword");
```

Comment:
It's not good practice. However, I think that we should support it, and somehow output a WARN if we find it in CConf rather than SConf, like a deprecation. It's best not to break properly configured clusters on upgrade.